src/cf-base.less

```diff
@@ -315,39 +315,57 @@ figure {
 */
 
 a {
-    border-width: 0;
-    border-style: dotted;
-    border-color: @pacific;
     color: @pacific;
     text-decoration: none;
+    
+    .cssgradients & {
+        background-image: linear-gradient(to right, fade(@pacific, 50%) 2px, transparent 2px);
+        background-size: 0 0;
+        background-repeat: repeat-x;
+        background-position: 0 1.0625em;
+    }
 
     &:visited,
     &.visited {
-        border-color: @teal;
         color: @teal;
+        
+        .cssgradients & {
+            background-image: linear-gradient(to right, fade(@teal, 50%) 2px, transparent 2px);
+        }
     }
 
     &:hover,
     &.hover {
-        border-style: solid;
-        border-color: @pacific-50;
         color: @pacific-50;
+        
+        .cssgradients & {
+            background-image: linear-gradient(to right, fade(@pacific-50, 50%) 2px, transparent 2px);
+          }
     }
 
     &:focus,
     &.focus {
-        border-style: solid;
         outline: thin dotted;
     }
 
     &:active,
     &.active {
-        border-style: solid;
-        border-color: @navy;
         color: @navy;
+        
+        .cssgradients & {
+            background-image: linear-gradient(to right, fade(@navy, 50%) 2px, transparent 2px);
+        }
     }
 }
 
+// To be documented: mixin for simulating underlines that break around descenders
```

Comment:
You've got a "To be documented" note here. Can you add that documentation please?